lib/python/Plugins/Extensions/FileCommander/plugin.py

```diff
@@ -2663,6 +2663,29 @@ def keyCancelCallback(self, answer):
 		if answer:
 			self.close()
 
+	def keyMenu(self):
+		def keyMenuCallback(answer):
+			match answer:
+				case "CLONE":
+					self.data.insert(self["data"].getCurrentIndex(), self["data"].getCurrent())
+				case "SORTA":
+					self.data.sort()
+				case "SORTD":
+					self.data.sort(reverse=True)
+				case "":
+					return
+			self["data"].setList(self.data)
+			self.isChanged = True
+
+		msg = [_("Please select the action for line number %d.") % self["data"].getCurrentIndex() + 1]
```

Comment:
msg = [_("Please select the action for line number %d.") % **(**self["data"].getCurrentIndex() + 1**)**]
TypeError: can only concatenate str (not "int") to str